### ISSUE TITLE
After installation, CNTK crashes my Python

### ISSUE THREAD

**Issue by skybe077** [Microsoft/CNTK#972]
![picture1](https://cloud.githubusercontent.com/assets/10797176/19829718/c67725fa-9e1b-11e6-963a-8e8ee4deae51.png)

As shown in the picture above. 

I've used manual installation.
Python doesn't crash if I change the **init**.py filename. 
But once that happens, the installation verification NumpyInterop/FeedForwardNet.py doesn't work at all. 

Any thoughts on how  I could go about it?
Thanks

**Comment by sayanpa** [comment]
Hi Skybe077,
Sorry to hear you having trouble. Would appreciate if you could give us a few more details. Did you install python first time? If not which version was installed? Can you please share the path variables? Are you able to activate the "CNTK-py34" environment? What changes are you making to init.py?

Thanks,
Skybe077

**Comment by skybe077** [comment]
Hiya,
Please see my answers for each question. 

1. Did you install python first time? If not which version was installed? 
I reinstalled it. It's version 3.4.3

2. Can you please share the path variables? 
I took this from command prompt. >> path 
is this what you're referring to?

PATH=C:\Users\User\Anaconda3\envs\cntk-py34;C:\Users\User\Anaconda3\envs\cntk-py
34\Library\mingw-w64\bin;C:\Users\User\Anaconda3\envs\cntk-py34\Library\usr\bin;
C:\Users\User\Anaconda3\envs\cntk-py34\Library\bin;C:\Users\User\Anaconda3\envs\
cntk-py34\Scripts;C:\Program Files\Microsoft MPI\Bin\;C:\ProgramData\Oracle\Java
\javapath;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS
Client\;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\Syste
m32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\Intel(R) Management Eng
ine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\
DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Pr
ogram Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Int
el\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program File
s\Lenovo\Touch Fingerprint Software\;C:\Program Files (x86)\Common Files\Lenovo;
C:\ProgramData\Lenovo\ReadyApps;C:\Program Files (x86)\Skype\Phone\;C:\Android;C
:\Windows\System32;C:\Program Files\Git\cmd;C:\Program Files (x86)\NVIDIA Corpor
ation\PhysX\Common;C:\Users\User\Anaconda3;C:\Users\User\Anaconda3\Scripts;C:\Us
ers\User\Anaconda3\Library\bin;;C:\Program Files\Docker Toolbox

3. Are you able to activate the "CNTK-py34" environment? 
No. It only works after I've renamed __init.py___ (pls see pic below)

4. What changes are you making to init.py?
I've renamed file to **a**_init.py. See picture. 

![picture1](https://cloud.githubusercontent.com/assets/10797176/19834107/df9df132-9e8c-11e6-8239-86f7152732d0.png)

**Comment by mahilleb-msft** [comment]
Hey @skybe077,

  perhaps Python versions are mixed. From your logs and screenshots, it is strange that you're running with the root Anaconda3 environment (c:\users\user\Anaconda3), however, in your path, there's your CNTK Python environment (...\envs\CNTK-py34) in addition.

  What Anaconda3 version did you install (I'm assuming 4.1.1)? Also, what OS version are you running on and which binary download are you using? Also, do you have an NVidia card?

  Let's in addition try this:
  Please revert all your file edits.
  Then, from a command prompt , execute the following steps and report the first step that fails (and output up to that point):
c:\users\user\anaconda3\scripts\activate c:\users\user\anaconda3\envs\cntk-py34
path
python --version
python -c "import cntk"

  If everything worked so far, try one of the validation samples.

  Thanks and sorry for the inconvenience,

  Mark

**Comment by wolfma61** [comment]
there have been many changes and improvement since this issue was created, If this still occurs with the most recent released version, please reopen (or create a new issue) with sufficient information to reproduce the problem

**Comment by Ryuk03** [comment]
I run into the same problem
![image](https://user-images.githubusercontent.com/8087746/38583404-86d75566-3cd8-11e8-84de-a77a115c3cc8.png)

the environment I have:
GeForece GTX1080
Driver version 391.35
CUDA v9.0
cudnn-9.0-v7.0
python 3.5.5 in Anoconda(conda 4.5.0)
![image](https://user-images.githubusercontent.com/8087746/38583727-89ace7aa-3cd9-11e8-807d-63fe5c31c094.png)

There is another problem that when I try `dir C:\Program Files\NVIDIA Corporation\NVSMI\nvml.dll` it shows the error that "The system cannot find the file specified." but the file existed in the folder.

**Comment by thiagocrepaldi** [comment]
Try dir "C:\Program Files\NVIDIA Corporation\NVSMI\nvml.dll" instead, with double quotes on the path with spaces

Also, run 'git pull' on your CNTK's master branch to get the commit 6e7ff0e974dd3323167e11062850f8cb9b0d9f2b. Validation messages were added to help debugging missing libraries

Finally, double check each folder you added to your PATH exists and has DLLs on it. Sometimes the path from the documentation slightly differs from the actual zip files

**Comment by Ryuk03** [comment]
After I delete the cudnn file that I extracted to the `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0` previously, and rerun the prompt again, the python works fine as below:
`
(cntk-py35) D:\Project\CNTK-Samples-2-5\Tutorials\NumpyInterop>python FeedForwardNet.py
Selected GPU[0] GeForce GTX 1080 as the process wide default device.
-------------------------------------------------------------------
Build info:

                Built time: Mar 15 2018 17:19:41
                Last modified date: Thu Mar 15 16:09:26 2018
                Build type: Release
                Build target: GPU
                With ASGD: yes
                Math lib: mkl
                CUDA version: 9.0.0
                CUDNN version: 7.0.4
                Build Branch: HEAD
                Build SHA1: 545a592f8399c4597c2587347a8b6bfb67cf07cd
                MPI distribution: Microsoft MPI
                MPI version: 7.0.12437.6
-------------------------------------------------------------------
Learning rate per minibatch: 0.5
 Minibatch[   1- 128]: loss = 0.614628 * 3200, metric = 29.25% * 3200;
 Minibatch[ 129- 256]: loss = 0.332382 * 3200, metric = 13.31% * 3200;
 Minibatch[ 257- 384]: loss = 0.298804 * 3200, metric = 11.59% * 3200;
 Minibatch[ 385- 512]: loss = 0.271270 * 3200, metric = 10.44% * 3200;
 Minibatch[ 513- 640]: loss = 0.247776 * 3200, metric = 9.03% * 3200;
 Minibatch[ 641- 768]: loss = 0.231605 * 3200, metric = 9.06% * 3200;
 Minibatch[ 769- 896]: loss = 0.229529 * 3200, metric = 8.62% * 3200;
 Minibatch[ 897-1024]: loss = 0.214106 * 3200, metric = 8.25% * 3200;
Finished Epoch[1]: loss = 0.305012 * 25600, metric = 12.45% * 25600 1.678s (15256.3 samples/s);
 error rate on an unseen minibatch 0.040000
`
Besides, the command works after add double quote.
Thanks

**Comment by warpdriv** [comment]
After reading almost everything to troubleshoot my same issuse, the solution for me was to upgrade my display driver since I'm running GPU installation and CUDA 9.2.

This page, https://docs.nvidia.com/deeplearning/sdk/cudnn-install/index.html#install-windows, has a section on:

One of the following supported CUDA versions and NVIDIA graphics driver:
NVIDIA graphics driver R377 or newer for CUDA 8
NVIDIA graphics driver R384 or newer for CUDA 9
NVIDIA graphics driver R390 or newer for CUDA 9.2

HTH
